integration_tests/steps/integration_tests.py

```diff
@@ -569,6 +569,31 @@ def test_output(context, scenario, sim_duration, slot_length, tick_length):
         assert all([permanent_load.strategy.energy == ei for ei in energy_profile])
 
 
+@then('we test energy bills')
+def test_energy_bills(context):
+    bills = context.simulation.endpoint_buffer.bills
+
+    cell_tower = bills["Cell Tower"]["sold"] - bills["Cell Tower"]["bought"]
+
+    house1 = bills["House 1"]["sold"] - bills["House 1"]["bought"]
+    area_net_bill = 0
+    for k, v in bills["House 1"]["children"].items():
```

Comment:
This for loop can be replaced with 
`area_net_bill = sum([v["sold"] - v["bought] for k, v in bills["House 1"]["children"].items()])`